src/test/java/org/openmrs/contrib/qaframework/RunTest.java

```diff
@@ -44,5 +45,7 @@
 				+ REGISTRATION;
 		public static final String SELENIUM_USER_ACCOUNT = SELENIUM + " and "
 				+ USER_ACCOUNT;
+		public static final String SELENIUM_FINDPATIENT = SELENIUM + " and "
```

Comment:
What do you think of `SELENIUM_FIND_PATTIENT`?

Reply:
Resolved thanks